pkg/model/external_access.go

```diff
@@ -42,7 +42,7 @@ func (b *ExternalAccessModelBuilder) Build(c *fi.ModelBuilderContext) error {
 	}
 
 	// SSH is open to AdminCIDR set
-	if b.Cluster.IsTopologyPublic() {
+	if b.Cluster.Spec.Topology.Masters == kops.TopologyPublic {
 		for i, sshAccess := range b.Cluster.Spec.SSHAccess {
 			c.AddTask(&awstasks.SecurityGroupRule{
 				Name:          s("ssh-external-to-master-" + strconv.Itoa(i)),
```

Comment:
Changed to `Name:          s("ssh-external-to-node-" + sshAccess),`  We have to be careful because this changes the terraform output, which will lead to spurious diffs, but I think we're already changing it in 1.5.

Sanitization of the name (e.g. for terraform) is done when we output to terraform